pretest.js

```diff
@@ -26,7 +26,7 @@ if (!process.env.MSSQL_DATABASE) process.env.MSSQL_DATABASE = global.getConfig()
 const catFileCmd = (isWin ? 'type ' : 'cat ') + sqlFileDir;
 
 const sqlcmd = catFileCmd + ' | ' + sqlDependencyDir + ' -s ' + process.env.MSSQL_HOST + ' -o ' + process.env.MSSQL_PORT +
-' -u ' + process.env.MSSQL_USER + ' -p ' + process.env.MSSQL_PASSWORD + ' -d ' + process.env.MSSQL_DATABASE;
+' -u ' + process.env.MSSQL_USER + ' -p "' + process.env.MSSQL_PASSWORD + '" -d ' + process.env.MSSQL_DATABASE;
```

Comment:
Thanks for contributing more fixes! Though maybe we should put this as a separate PR to keep things clean?

Reply:
Ready